docs/src/main/asciidoc/kubernetes-dev-services.adoc

```diff
@@ -0,0 +1,55 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services for Kubernetes
+include::_attributes.adoc[]
+:categories: messaging
```

Comment:
A little late for this but I suspect this category isn't correct…

Reply:
@metacosm I will have a look thanks

Reply:
My bad. Should be cloud I guess like other k8s pages.